src/commands/generate.rs

```diff
@@ -9,18 +9,36 @@ pub fn generate(
     name: &str,
     template: &str,
     target_type: Option<TargetType>,
+    site: bool,
+) -> Result<(), failure::Error> {
+    let target_type = target_type.unwrap_or_else(|| get_target_type(template));
+    if site {
+        run_generate(name, template, &target_type)?;
+        let config_path = PathBuf::from("./").join(&name);
+        Manifest::generate(name.to_string(), target_type, config_path, site)?;
+        Ok(())
+    } else {
+        run_generate(name, template, &target_type)?;
+        let config_path = PathBuf::from("./").join(&name);
+        Manifest::generate(name.to_string(), target_type, config_path, site)?;
+        Ok(())
```

Comment:
maybe i'm silly... but why don't we scaffold a site here? I would have thought the scaffolding would happen in `generate` and not `init` (since I thought `init` just introduces a `wrangler.toml`)

Reply:
it's a bit wacky BUT, scaffold only puts the contents of the workers-site directory (and the directory itself). generate does a full project directory from a github repo, and doesn't remove the git project.

Reply:
gotcha, this makes more sense now. let me make sure wrangler generate also calls scaffolding function.

Reply:
i think there's a better way to do it, but at the moment this is kinda what i got